docs/usage/configuration-options.md

```diff
@@ -702,6 +702,16 @@ It is pointless to edit the labels, as Renovate bot restores the labels on each
 
 Configure this option if you prefer a different title for the Dependency Dashboard.
 
+## dependencyDashboardOSVVulnerabilitySummary
+
+Set this function to control if a summary of the CVEs affected should be added to the dependency dashboard.
+
+Available options are:
+
+- `none` ( default ) no section will be added.
+- `unresolved` only vulnerabilities will be added if there are no fixes available.
+- `all` list all vulnerabilities which affect this repository.
```

Comment:
```suggestion
Use this option to control if the Dependency Dashboard should have a section that shows the CVEs for your repository.

The source of these CVEs is EDIT SUGGESTION TO INSERT LINK HERE.

You can choose from:

- `none` (default) no section will be added
- `unresolved` show vulnerabilities that have no fixes
- `all` list all vulnerabilities
```

Changes:

- Rewrite
- Drop punctuation at end of list items
- Use simple words and phrases
- Add draft text that explains the source of the CVEs (you need to fill this out)